lsp-docker.el

```diff
@@ -550,16 +557,12 @@ Argument DOCKER-CONTAINER-NAME name to use for container."
 
 (cl-defun lsp-docker--build-image-and-register-server-async (&key image-name
                                                                   dockerfile-path
-                                                                  project-root
```

Comment:
Please let me know what you think :)

Reply:
> I left them in place only because there were future plans for them

If you could share it I can propose the following trade off:
- restore the arguments but left them commented out
- add a TODO comment on top describing the future plans and that those should be uncommented when the time is right for implementing the features you had in mind.

I believe this is better than leaving them yet not being used, I found it pretty confusing. Consider also in that in the next feature implementation steps I imagine they might be dragged around.

In general I always try to document anything that is not obvious, there are countless contributors out there, the code and the comments are few of those things that keep us interconnected :smile:

Reply:
The original idea (for example) was to use `project-root` inside a Dockerfile when building the image (probably as an environment variable).

I propose we just comment them out :) We can probably even write a small comment explaining why